code-formatter/code_formatter.rb

```diff
@@ -16,7 +16,7 @@ def run
 
   def format_terraform_code
     terraform_directories_in_pr.each do |dir|
-      executor.execute "terraform fmt #{dir}"
+      executor.execute "terraform fmt -recursive #{dir}"
```

Comment:
If this is recursive, you could ditch the `#{dir}` variable as it will recurse through the subdirectories?

Reply:
sure let me have a look

Reply:
@jakemulley done